packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -217,7 +228,12 @@ contract L2OutputOracle is Initializable, Semver {
             );
         }
 
-        emit OutputProposed(_outputRoot, nextOutputIndex(), _l2BlockNumber, block.timestamp);
+        // Post the bond to the bond manager
+        BOND_MANAGER.post{ value: msg.value }({
+           _bondId: bytes32(abi.encode(_l2BlockNumber)),
+           _bondOwner: msg.sender,
+           _minClaimHold: FINALIZATION_PERIOD_SECONDS
```

Comment:
An invariant exists that there must always be a bond while the output is mutable (able to be removed)